test/fixtures/className merging/code.js

```diff
@@ -0,0 +1,2 @@
+import "../../macro"
```

Comment:
Please do not use whitespace and only lowercase in filenames.

in some project we had problems because of case-insensitive vs case-sensitive file systems.